functional/webhook-certificate-on-localhost/test.sh

```diff
@@ -20,33 +20,33 @@ rlJournalStart
             done
         fi
 
-        # restart journald as it seems to get stuck on RHEL-10 but
-        # I wasn't able to figure out the details yet
-        rlRun "systemctl restart systemd-journald "
-
         # Create directory to store certificates and keys
         CERTDIR=/var/lib/keylime/certs
         rlRun "mkdir -p $CERTDIR"
 
         # Generate keys for TLS certificates
         rlRun "x509KeyGen good-ca" 0 "Generating good CA RSA key pair"
         rlRun "x509KeyGen bad-ca" 0 "Generating bad CA RSA key pair"
+        rlRun "x509KeyGen installed-ca" 0 "Generating bad CA RSA key pair"
         rlRun "x509KeyGen intermediate-ca" 0 "Generating Intermediate CA RSA key pair"
         rlRun "x509KeyGen verifier" 0 "Generating verifier RSA key pair"
         rlRun "x509KeyGen verifier-client" 0 "Generating verifier-client RSA key pair"
         rlRun "x509KeyGen registrar" 0 "Generating registrar RSA key pair"
         rlRun "x509KeyGen tenant" 0 "Generating tenant RSA key pair"
         rlRun "x509KeyGen good-webhook" 0 "Generating webhook RSA key pair"
         rlRun "x509KeyGen bad-webhook" 0 "Generating bad webhook RSA key pair"
+        rlRun "x509KeyGen installed-webhook" 0 "Generating bad webhook RSA key pair"
```

Comment:
Same as above, Fix the message to mention installed instead of bad

Reply:
also fixed